Content.Shared/Implants/SharedSubdermalImplantSystem.cs

```diff
@@ -54,6 +55,11 @@ private void OnInsert(EntityUid uid, SubdermalImplantComponent component, EntGot
                 }
             }
         }
+
+        if (_tag.HasTag(uid, "MindShield"))
+        {
+            EnsureComp<MindShieldComponent>(component.ImplantedEntity.Value);
+        }
```

Comment:
This shouldn't be here, if you really need comps being added use compregistry though I doubt you do.

Reply:
Well I use this comp to prevent conversion as well as remove conversion so I kind of need this here as I've got no idea where to put it.

Reply:
This is the system for subdermal implants, not mindshields.

MindshieldSystem should be the one handling this via event or otherwise.